frontend/src/components/Forms/TeamEventCreditsForm/TeamEventCreditsForm.tsx

```diff
@@ -75,11 +76,18 @@ const TeamEventCreditForm: React.FC = () => {
         pending: true,
         uuid: ''
       };
+
       requestTeamEventCredit(newTeamEventAttendance).then(() => {
-        setPendingAttendance((pending) => [...pending, newTeamEventAttendance]);
-        Emitters.generalSuccess.emit({
-          headerMsg: 'Team Event Credit submitted!',
-          contentMsg: `The leads were notified of your submission and your credit will be approved soon!`
+        TeamEventsAPI.getTeamEventAttendanceByUser().then((attendance) => {
```

Comment:
Sorry, not to keep adding nits, but we can do this more efficiently by using the `attendance` object that we get from `requestTeamEventCredit`. i.e. 

```
requestTeamEventCredit(newTeamEventAttendance).then((createdAttendance) => {
```